src/lib/Microsoft.Fx.Portability/IApiPortOptions.cs

```diff
@@ -12,6 +12,8 @@ public interface IApiPortOptions
         string Description { get; }
 
         /// <summary>
+        /// Gets a dictionary of input assembly files and whether or not they
+        /// were specified or not.
```

Comment:
"... and whether they were specified." ?

Reply:
Fixed